apps/docs/tests/cypress.config.ts

```diff
@@ -6,11 +6,10 @@ export default defineConfig({
   viewportWidth: 1920,
   defaultCommandTimeout: 20000,
   video: false,
-  
   e2e: {
     baseUrl: 'http://localhost:8080/v2',
     specPattern: '../tests/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: '../tests/support/index.ts',
   },
-  chromeWebSecurity: false
+  chromeWebSecurity: false,
```

Comment:
Is that needed?

Reply:
auto formatting after removal empty line, so not required but welcomed :)

Reply:
I meant `chromeWebSecurity: false`

Reply:
This is soemthing I wasnt done myself but I will check it withing CI setup